src/Sarif.Driver/Sdk/SarifLogger.cs

```diff
@@ -102,7 +102,7 @@ public static ToolInfo CreateDefaultToolInfo(string prereleaseInfo = null)
                             }
                             });
                         }
-                        runInfo.AnalysisTargets.Add(fileReference);
+                        runInfo.FileInfo.Add(fileReference.Uri.OriginalString, new FileReference[] { fileReference });
```

Comment:
... and `FileInfo` is a dictionary from strings to arrays of `FileReference` (each element of which corresponds to a file nesting); it's no longer just an array of `FileReference`.